### PR TITLE
Rocks db window service

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
Current implementation of window service doesn't use new db_window module

#### Summary of Changes
1) Integrate RocksDb into window service
2) Fix associated tests

Fixes #

### CODE REVIEW

File: src/bin/replicator.rs [hunk 1]
```diff
@@ -94,7 +95,14 @@ fn main() {
...
     let entry_height = 0;
 
+    // Create the RocksDb ledger, eventually will simply repurpose the input
+    // ledger path as the RocksDb ledger path
+    let db_ledger_path = format!("{}/{}", ledger_path.unwrap(), DB_LEDGER_DIRECTORY);
+    let db_ledger = Arc::new(RwLock::new(
+        DbLedger::open(&db_ledger_path).expect("Expected to be able to open database ledger"),
+    ));
     let (replicator, leader_info) = Replicator::new(
+        db_ledger,
```
Comment (rob-solana) [inline src/bin/replicator.rs]: I think it'd be better if you passed the ledger_path as the argument, did the opening in src/fullnode.rs.  This way we can move to sled without changing this code.

Comment (carllin) [reply src/bin/replicator.rs]: I think this comment was meant for bin/fullnode.rs? I made the change there :)

Comment (rob-solana) [reply src/bin/replicator.rs]: is the change not appropriate here, as well?

Comment (carllin) [reply src/bin/replicator.rs]: The replicator doesn't open a fullnode, but I can move the open logic to src/replicator.rs to parallel the change in src/fullnodes.rs

**Pull request merged**